spec/system/kudo_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'Kudo test' do
+  before do
+    driven_by(:rack_test)
+  end
+
+  it 'Creates, update and destroy kudo' do
```

Comment:
also here it is nice practice to test only one thing in one spec, so i would suggest to create three tests:
`it 'creates kudo' do`
`it 'edits kudo' do`
`it 'removes kudo' do`

what do you think about it?

Reply:
I think that by dividing one kudo test into 3 smaller ones, it gains readability and understanding